tests/Command/ChangePasswordCommandTest.php

```diff
@@ -94,10 +97,37 @@ public function testChangePassword()
 
         $this->authenticationManager->shouldReceive('update')->with($authentication);
 
-        $this->commandTester->execute(array(
-            'command'      => self::COMMAND_NAME,
-            'userId'         => '1'
-        ));
+        $this->commandTester->execute(array('command' => self::COMMAND_NAME, 'userId' => '1'));
+
+
+        $output = $this->commandTester->getDisplay();
+        $this->assertRegExp(
+            '/Password Changed/',
+            $output
+        );
+    }
+
+
+    public function testChangePasswordWOnLegacyAccount()
```

Comment:
Something up with this test title. `testChangePasswordOnLegacyAccount`?

Reply:
corrected.